### PR TITLE
Upgrade to latest sphinx 3.3.0

### PR DESCRIPTION
**Pull request by wendigo**
Requires released version of https://github.com/prestosql/docker-images/pull/77

### CODE REVIEW

File: presto-docs/build [hunk 1]
```diff
@@ -6,7 +6,7 @@ cd "${BASH_SOURCE%/*}"
 
 test -t 1 && OPTS='-it' || OPTS=''
 
-SPHINX_IMAGE=${SPHINX_IMAGE:-prestodev/sphinx:2}
+SPHINX_IMAGE=${SPHINX_IMAGE:-prestodev/sphinx:latest}
```
Comment (electrum) [inline presto-docs/build:9]: We should continue to depend on an exact version so that builds are deterministic.

**Comment by electrum** [comment]
Thanks for updating Sphinx. This is something I've been wanting to do for a while. However, I don't think we should move the Docker image out now. I'm happy to merge this now without that part of the change.

**Review by mosabua — APPROVED** [review]
Looks good. I like the removal of the duplicate version as well!

**Review by electrum — APPROVED** [review]

**Comment by electrum** [comment]
Thanks!

**Pull request merged**